searx/webapp.py

```diff
@@ -877,8 +877,21 @@ def __get_translated_errors(unresponsive_engines: Iterable[UnresponsiveEngine]):
 
 @app.route('/about', methods=['GET'])
 def about():
-    """Render about page"""
-    return render('about.html', help=user_help.HELP)
+    """Redirect to about page"""
+    return redirect(url_for('help_page', pagename='about'))
```

Comment:
Note: The link in the base.html templates can be `{{ url_for('help_page', pagename='about') }}` to avoid a redirect. It can be in a second time since the link may change again with the i18n support.